internal/db/migrations/000023_add_derivation_to_joyid_infos.up.sql

```diff
@@ -0,0 +1,7 @@
+ALTER TABLE joy_id_infos ADD COLUMN `derivation_c_id` varchar(1500) NOT NULL DEFAULT '' AFTER `cota_cell_id`,
+                        ADD COLUMN `derivation_commitment` varchar(64) NOT NULL DEFAULT '' AFTER `derivation_c_id`;
```

Comment:
The commitment length is 4 bytes, why use `varchar(64)` ?

Reply:
This requires simultaneously supporting both the main key and sub key. The commit(secret) length of the sub key is 32 bytes.

Reply:
There may be changes in the future here to avoid repetitive adjustments